external/gtest/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 2.8.8)
```

Comment:
Did you mean to check in this file?  Everything in the external directory (currently) is downloaded dynamically at build time.